### ISSUE TITLE
School clubs Periods not found

### ISSUE THREAD

**Issue by 0Charliecat** [loumadev/EdupageAPI#11]
hey, sorry me again. This is probably error on Edupage's side. Got this error.

```
#
# An error has occurred!
#
# This is probably not your fault!
# It should not cause the program to crash, but some functionality can be limited.
# If this happens often, please consider reporting a bug.
# You can disable the warnings by including "--edu-disable-warnings=true" as a CLI argument.
#
# Error thrown:
#  ReferenceError: Failed to find period for lesson
#      at Lesson.init (/home/runner/edupejc/node_modules/edupage-api/src/Lesson.js:132:27)
#      at new Lesson (/home/runner/edupejc/node_modules/edupage-api/src/Lesson.js:109:42)
#      at /home/runner/edupejc/node_modules/edupage-api/src/Timetable.js:57:78
#      at Array.map (<anonymous>)
#      at Timetable.init (/home/runner/edupejc/node_modules/edupage-api/src/Timetable.js:57:66)
#      at new Timetable (/home/runner/edupejc/node_modules/edupage-api/src/Timetable.js:47:45)
#      at /home/runner/edupejc/node_modules/edupage-api/src/Edupage.js:467:68
#      at Array.map (<anonymous>)
#      at Edupage.fetchTimetablesForDates (/home/runner/edupejc/node_modules/edupage-api/src/Edupage.js:467:43)
#
# Summary:
#  Time: Mon Dec 13 2021 20:45:49 GMT+0000 (Coordinated Universal Time)
#  EdupageAPI version: v0.7.13
#  Node.js version: v16.7.0
#  Error log: /home/runner/edupejc/node_modules/edupage-api/logs/warning_2021-12-13T20-45-49Z.json
#
# If you would like to submit an issue, please visit:
#  > https://github.com/loumadev/EdupageAPI/issues
#  (Please, include this error message + error log)
#
```

im ignoring it now im just letting you know. there is an error with school clubs
[warning_2021-12-13T20-34-01Z.json (log file)](https://github.com/loumadev/EdupageAPI/files/7706897/warning_2021-12-13T20-34-01Z.json.txt)

some things are redacted for sake of my own privacy

have a nice day!

**Comment by 0Charliecat** [comment]
There's a `starttime` and `endtime`
but there isn't period set,

![image](https://user-images.githubusercontent.com/50016870/145887661-78964109-7a40-4f03-9843-d982e14e4bf5.png)

**Comment by loumadev** [comment]
Hello, great catch. I've published a new version [(v0.7.14)](https://github.com/loumadev/EdupageAPI/releases/tag/v0.7.14), so please, update the library and try it again. It should now create a brand new `Period` object, filled with "invalid" data. It uses available data to construct at least something, so your `startTime` and `endTime` should be included as expected.
